### PR TITLE
Adds DocKing to Document Management

### PR DESCRIPTION
**Pull request by sethsandaru**
Hi team, DocKing is 4+ months old and I'd like to add this to our lovely list of self-hosted software.

DocKing is free, fully open-sourced. Helps us to centralize document templates & handle render/export PDFs dynamically.

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/docking.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: DocKing
+website_url: https://docking.shipsaas.tech
+description: Your shared-microservice that takes over the document templates management & render/export PDF dynamically.
+licenses:
+  - MIT
+platforms:
+  - PHP
+  - Javascript
+  - Docker
```
Comment (KieranRobson) [inline software/docking.yml:9]: > https://docking.shipsaas.tech/getting-started/run-on-local/normal-run

```suggestion
platforms:
  - Nodejs
  - PHP
  - Docker
```

File: software/docking.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: DocKing
+website_url: https://docking.shipsaas.tech
+description: Your shared-microservice that takes over the document templates management & render/export PDF dynamically.
```
Comment (KieranRobson) [inline software/docking.yml:3]: Cleaner description

```suggestion
description: Document management service/microservice that handles templates and renders them in PDF format, all in one place.
```

File: software/docking.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  - PHP
+  - Javascript
+  - Docker
+tags:
+  - Document Management
+source_code_url: https://github.com/shipsaas/docking
+demo_url: https://docking-demo.shipsaas.tech/console
+stargazers_count: 69
+updated_at: '2023-10-29'
+archived: false
```
Comment (KieranRobson) [inline software/docking.yml:16]: This is all autofilled so we can get rid of it for now.

```suggestion
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Couple changes.

File: software/docking.yml (hunk 3 above)
Comment (sethsandaru) [reply software/docking.yml:16]: cool, good to know, removed

File: software/docking.yml (hunk 2 above)
Comment (sethsandaru) [reply software/docking.yml:3]: cool, updated

File: software/docking.yml (hunk 1 above)
Comment (sethsandaru) [reply software/docking.yml]: updated thanks

**Review by KieranRobson — APPROVED** [review]
Looks good to me - Thanks!

**Review by nodiscc — APPROVED** [review]

**Pull request merged**